src/components/mobile-menu-container/components/mobile-user-disclosure/index.tsx

```diff
@@ -50,11 +50,15 @@ const renderItem = (
 	return <li key={index}>{content}</li>
 }
 
-const MobileUserDisclosure = ({ items, user }: MobileUserDisclosureProps) => {
+const MobileUserDisclosure = ({
+	items,
+	user,
+	initialOpen,
+}: MobileUserDisclosureProps) => {
 	const { icon, description } = getUserMeta(user)
 
 	return (
-		<Disclosure containerClassName={s.root}>
+		<Disclosure containerClassName={s.root} initialOpen={initialOpen}>
```

Comment:
The designs call for this disclosure to be already open when the mobile menu is opened. The way I have it now, this is drilled down from `SidebarSidecarLayout` 🤷‍♀️  Open to alternative approaches.

Reply:
🤔  good idea, yeah that does feel better than this approach. ill update.

Reply:
I updated `MobileAuthenticatedControls` to handle this 👌 Feels better to me!